core/src/main/resources/swagger/api-docs.yaml

```
@@ -5625,6 +5625,9 @@ components:
         workspaceVersion:
           type: string
           description: internal use
+        currentBillingAccountOnGoogleProject:
+          type: string
+          description: The current project the workspace is being billed to. For desired billing account, look to the workspace's billing project
```

Sorry for the nit, but I would say something like:  
```suggestion
          description: The current billing account being charged for activity within this workspace. For desired billing account, look to the billing account specified on this workspace's billing project
```
I am really sorry for being so picky on this, but as I've been digging into this area of Rawls, I'm finding it really helpful to always try to be as precise as possible.   For example, when someone says "project" do they mean Terra Billing Project or a Google Project?  When they saying "billing", do they mean Terra Billing Project or Google Billing Account?  It's just so easy to get confused, I'd rather over-explain in order to avoid ambiguity.